src/GraphQL/SchemaExtensions.cs

```diff
@@ -7,6 +7,7 @@ namespace GraphQL
 {
     public static class SchemaExtensions
     {
+        [Obsolete(@"This action is async. Use ExecuteAsync. Will be removed in v4.")]
```

Comment:
Let`s remove this method. Is there any point in trying to maintain  such "local" backward compatibility if backward compatibility has already been lost for a long time for version 3.0.0?

Reply:
done